.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy Documentation
+# This job builds and deploys documenation to github pages.
+# It runs on every push to master.
+on:
+  push:
+    branches:
+      - main
+  schedule:
```

Comment:
we can now change this to every time we modify the docs directory.

Reply:
True, changed :+1: